src/components/users/teachers/LogForm.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import Line from "./Line";
+
+export default function LogForm({ cohorts, logsData, setLogsData }) {
```

Comment:
Is this FORM for UPDATE or CREATE or BOTH?

If both, I'd rename the form to DeliveryLogCreateUpdateForm